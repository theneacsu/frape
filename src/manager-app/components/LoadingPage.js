import React from 'react'

const LoadingPage = () => (
  <div className="lds-heart">
    <div />
  </div>
)

export default LoadingPage
