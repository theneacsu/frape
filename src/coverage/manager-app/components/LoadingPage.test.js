import React from 'react'
import { shallow } from 'enzyme'
import LoadingPage from '../../../manager-app/components/LoadingPage'

test('Should render correctly the LoadingPage component', () => {
  const wrapper = shallow(<LoadingPage />)
  expect(wrapper).toMatchSnapshot()
})
