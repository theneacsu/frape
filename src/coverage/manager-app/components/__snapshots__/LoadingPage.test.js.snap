// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly the LoadingPage component 1`] = `
<div
  className="lds-heart"
>
  <div />
</div>
`;
